### ISSUE TITLE
JSDoc

### ISSUE THREAD

**Issue by shmygol** [mahaker/esbuild-gas-plugin#20]
It would be nice if JSDoc were preserved to enable autocompletion for custom functions.

**Comment by mahaker** [comment]
Generally agreed!
I'm waiting for your PullRequest!

**Comment by shmygol** [comment]
I don't have much experience with esbuild, but it doesn't seem to be possible: https://github.com/evanw/esbuild/issues/294.

**Comment by mahaker** [comment]
@shmygol 
Sorry, I understand what you are requesting.
You want this plugin to also output JSDoc for functions implemented by the user.
Is this correct?

**Comment by shmygol** [comment]
Correct. It is essential, because custom formulas in Google Spreadsheets need JSDoc to support auto-completion and user documentation. Instead of using `esbuild-gas-plugin` I ended up bundling the business logic with an esbulid parameter `glabalName: 'Main'` and push the bundle together with an unmodified custom `index.js` containing formulas and hooks required by GAS like this: 

```
/**
 * Hello World
 *
 * @param {string} name Your name
 * @return {string}
 * 
 * @customfunction
 */
function HELLO(name) {
    return Main.hello(name);
}
```

**Comment by mahaker** [comment]
@shmygol 
Sorry for the delay in responding.

New version v0.10.0 has been released.
https://github.com/mahaker/esbuild-gas-plugin/releases/tag/0.10.0

It includes support for JSDoc.
Please give it a try!
